EncoreTickets.SDK/Inventory/IInventoryServiceApi.cs

```diff
@@ -86,7 +86,7 @@ public interface IInventoryServiceApi : IServiceApi
         /// <param name="quantity">quantity of seats needed</param>
         /// <param name="performance">performance: if nothing is sent, current time will be used</param>
         /// <returns>Array of availability seat</returns>
-        [Obsolete]
+        [Obsolete("The data returned by this method is not compatible with the latest basket service.")]
```

Comment:
I think it would make more sense to include the name of the method that we recommend to use instead of this one into the message.